### PR TITLE
fixed: do not include headers into Opm namespace

### PR DESCRIPTION
**Pull request by akva2**
This gave me some headaches..

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/simulators/wells/GasLiftStage2_impl.hpp [hunk 1]
```diff
@@ -1140,4 +1142,4 @@ updateRates(const std::string &well_name)
     this->alq += delta_alq;
 }
 
-
+}
```
Comment (bska) [inline opm/simulators/wells/GasLiftStage2_impl.hpp:1145]: Would you mind tagging this closing brace with `// namespace Opm`?

Comment (akva2) [reply opm/simulators/wells/GasLiftStage2_impl.hpp]: done.

**Comment by akva2** [comment]
jenkins build this please

**Comment by bska** [comment]
Build check is green.  I'll merge into master.

**Comment by hakonhagland** [comment]
> This gave me some headaches

@akva2 Why? I thought I was being clever here :smile: By including the implementation header at that point I thought I avoided having the `namespace Opm { .. }` in the implementation file. But maybe it is not so readable if you are used to having the `namespace Opm {...}` in the implementation?

**Comment by akva2** [comment]
Because you gor Opm::fmt namespace and Opm::Opm namespace for those headers you included on top of the _impl file. you were saved by header guards and the fact that all those were already included but i am reshuffling things so that was no longer the case.

**Comment by hakonhagland** [comment]
> Because you got Opm::fmt namespace and Opm::Opm namespace for those headers you included on top of the _impl file

@akva2  Yes. So then we would get `Opm::Opm` instead of `Opm` since the header was already inside `Opm`.. Thanks for clarifying.

**Pull request merged**